static/js/emoji_picker.js

```diff
@@ -67,13 +67,17 @@ function generate_emoji_picker_content(id) {
         var emojis_used = reactions.get_emojis_used_by_user_for_message_id(id);
         _.each(emojis_used, function (emoji_name) {
             var is_realm_emoji = emojis[emoji_name].is_realm_emoji;
+            var emoji_url = emojis[emoji_name].url;
             emojis[emoji_name] = {
                 name: emoji_name,
                 has_reacted: true,
-                css_class: emoji.emojis_name_to_css_class[emoji_name],
                 is_realm_emoji: is_realm_emoji,
-                url: emojis[emoji_name].url,
             };
+            if (is_realm_emoji) {
+                emojis[emoji_name].url = emoji_url;
+            } else {
+                emojis[emoji_name].css_class = emoji.emoji_name_to_css_class(emoji_name);
+            }
```

Comment:
`emoji_name_to_css_class()` function no longer exists. It was a redundant redirection, so I had removed it in a commit which has already been merged in the master. I have amended the commit and included it in my PR.